### PR TITLE
Script to get failing services from submissions (New)

### PR DESCRIPTION
**Pull request by fernando79513**
## Description

One of the most failed tests on the last SRU review was the power-management/post-warm-reboot test. These tests are failing because there are some services on some devices that are failing.
We want to get some more information about the tests that are failing, so we can in the future fix the most recurrent ones.

## Resolves

https://warthogs.atlassian.net/browse/CHECKBOX-1467

## Test

1. Create a `submissions.txt` file with the submissions you want to analyze:
```
https://certification.canonical.com/hardware/202212-31030/submission/370807/
https://certification.canonical.com/hardware/202301-31145/submission/371588/
https://certification.canonical.com/hardware/202212-30927/submission/371010/
```
2. Get your Session Id from your C3 profile under the Developer section
3. Run the script
```
python get_failing_services.py submissions.txt --session-id <YOUR_ID> --test post-warm-reboot
```

### CODE REVIEW

File: submissions-tools/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Submission Tools
+
+This folder contains some tools designed to interact with the certification
+website, specifically for downloading submissions and analyzing them.
+
+These script can be really useful when trying to investigate a test failure in
```
Comment (pieqq) [inline submissions-tools/README.md:6]:
```suggestion
These scripts can be really useful when trying to investigate a test failure in 
```

File: submissions-tools/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+   downloads the submissions, and saves them locally for further analysis.
+
+- **get_failing_services.py**: Downloads and analyzes a list of submissions.
+   Then, uses the test output files to identify any services that failed to
+   start post-reboot. It supports analyzing results from both warm and cold
+   reboots.
+
+## Usage
+
+## Prerequisites
```
Comment (pieqq) [inline submissions-tools/README.md:33]:
```suggestion
### Prerequisites
```

File: submissions-tools/README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+2. Run the `download_submissions.py` script, providing the path to your text
+   file and your session ID:
+
+```sh
+./download_submissions.py submissions_file.txt --session-id YOUR_SESSION_ID
+```
+
+3. The script will download the submissions and save them in a local directory
+   structure.
```
Comment (pieqq) [inline submissions-tools/README.md:59]:
```suggestion
3. The script will download the submissions and save them in a local directory
   structure. Submissions that have already been downloaded will be skipped.
```

**Review by pieqq — APPROVED** [review]
Thanks for all applying all the changes we discussed on MM!

Apart from a typo and a suggestion in the README, it's good to land.

**Review by fernando79513 — COMMENTED** [review]
Applied PR suggestions

**Pull request merged**